warehouse/query-core/src/main/java/datawave/query/function/FacetedGrouping.java

```diff
@@ -177,7 +177,7 @@ public FacetedGrouping(FacetedConfiguration config) {
         
         if (fixNewList.size() > config.getMaximumFacetGroupCount()) {
             // we've exceeded, so let's get a minimum adjustment factor
-            final int groupAdjustmentFactor = (int) Math.ceil(fixNewList.size() / config.getMaximumFacetGroupCount());
+            final int groupAdjustmentFactor = (int) Math.ceil((double) fixNewList.size() / config.getMaximumFacetGroupCount());
```

Comment:
[ERROR] Integral value cast to double and then passed to Math.ceil in datawave.query.function.FacetedGrouping.adjustAttributeGrouping(Collection, Set) [datawave.query.function.FacetedGrouping] At FacetedGrouping.java:[line 180] ICAST_INT_CAST_TO_DOUBLE_PASSED_TO_CEIL

Spotbugs caught several examples of this.